libs/ui/packages/react/src/components/carousel/footer/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { ItemStatus } from "./pagination/types";
+
+/**
+ * Returns the status of an indexed item from the carousel index.
+ */
+export const getItemStatus = (itemIndex: number, activeIndex: number) => {
+  const itemDistanceFromActiveIndex = Math.abs(itemIndex - activeIndex);
+
+  if (itemDistanceFromActiveIndex === 0) {
+    return ItemStatus.active;
+  } else if (itemDistanceFromActiveIndex === 1) {
+    return ItemStatus.nearby;
+  } else if (itemDistanceFromActiveIndex === 2) {
+    return ItemStatus.far;
+  }
```

Comment:
```suggestion
  }
  if (itemDistanceFromActiveIndex === 1) {
    return ItemStatus.nearby;
  }
  if (itemDistanceFromActiveIndex === 2) {
    return ItemStatus.far;
  }
```
You don't need the else as you early return